packages/lib/modules/swap/SwapProvider.tsx

```diff
@@ -502,7 +501,9 @@ export function _useSwap({ poolActionableTokens, pool, pathParams }: SwapProvide
   // Sets initial swap state for pool swap edge-case
   function setInitialPoolSwapState(pool: Pool) {
     const { tokenIn } = pathParams
-    setInitialChain(pool.chain)
+    const slugChain = chainToSlugMap[pool.chain]
```

Comment:
This was not causing a bug cause we had `if (isPoolSwapUrl) return` in `setSelectedChain` which becomes unnecessary with this fix.